web/src/test/java/com/graphhopper/http/GraphHopperServletIT.java

```diff
@@ -39,27 +40,30 @@
 /**
  * @author Peter Karich
  */
-public class GraphHopperServletIT extends BaseServletTester {
+public class GraphHopperServletIT {
     private static final String DIR = "./target/andorra-gh/";
 
-    @AfterClass
-    public static void cleanUp() {
-        Helper.removeDir(new File(DIR));
-        shutdownJetty(true);
-    }
+    private static final GraphHopperConfiguration config = new GraphHopperConfiguration();
 
-    @Before
-    public void setUp() {
-        CmdArgs args = new CmdArgs().
+    static {
+        config.cmdArgs = new CmdArgs().
                 put("config", "../config-example.properties").
                 put("datareader.file", "../core/files/andorra.osm.pbf").
                 put("graph.location", DIR);
-        setUpJetty(args);
+    }
+
+    @ClassRule
+    public static final DropwizardAppRule<GraphHopperConfiguration> app = new DropwizardAppRule(
+            GraphHopperApplication.class, config);
+
+    @AfterClass
+    public static void cleanUp() {
+        Helper.removeDir(new File(DIR));
     }
 
     @Test
     public void testBasicQuery() throws Exception {
-        JsonNode json = query("point=42.554851,1.536198&point=42.510071,1.548128", 200);
+        JsonNode json = app.client().target("http://localhost:8080/route?point=42.554851,1.536198&point=42.510071,1.548128").request().buildGet().invoke().readEntity(JsonNode.class);
```

Comment:
Yeeeeeeah, also it looked a bit like it was starting many servers but shutting down only one. 

I see the point, but please also see the transparency of just writing down the port number which is tested. And of testing the real server application with its real default port number / real way of configuring its port number.